geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionAlterFunction.java

```diff
@@ -150,6 +151,25 @@ public void execute(FunctionContext context) {
       }
     }
 
+    final ClassName<CustomExpiry> entryIdleCustomExpiry =
```

Comment:
Why do we need to make this variable `final`?  I appreciate the consistency with the surrounding code, but I don't see why any of those need to be made `final` either.  They are all single-use variable names already.

Reply:
I don't know either why they need to be final.